x-pack/plugins/security_solution/public/common/components/user_privileges/endpoint/use_endpoint_privileges.ts

```diff
@@ -61,7 +62,7 @@ export const useEndpointPrivileges = (): Immutable<EndpointPrivileges> => {
 
     const privilegeList: EndpointPrivileges = Object.freeze({
       loading,
-      ...(!loading && fleetAuthz
+      ...(!loading && fleetAuthz && !isEmpty(user)
```

Comment:
Check if current user exists (is authenticated), if not, returns initial state of endpoint authz so we can display the missing privileges pages/callouts/messages if needed.